.github/workflows/capsule-cypress-manual-trigger.yml

```diff
@@ -0,0 +1,138 @@
+name: Capsule tests -- manual trigger
+
+# This workflow runs the frontend tests against chosen branch
+
+on:
+  workflow_dispatch:
+    inputs:
+      project:
+        description: 'Project'
+        required: true
+        type: choice
+        options:
+          - explorer
+          - simple-trading-app
+          - stats
+          - token
+          - trading
+      includeFlows:
+        description: 'Include long flows (if any)?'
+        required: true
+        type: boolean
+        default: true
+
+jobs:
+  manual:
+    name: Run capsule tests -- manual trigger
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: 'on'
+      GOPROXY: ${{ secrets.GO_PROXY }}
+    steps:
+      #######
+      ## Setup langs
+      #######
+      - name: Set up Go
+        uses: actions/setup-go@v3
+        id: go
+        with:
+          go-version: 1.18
+      - name: Set up Node 16
+        uses: actions/setup-node@v2
+        id: npm
+        with:
+          node-version: 16
+
+      #######
+      ## Checkout capsule
+      #######
+
+      # Checkout capsule to build local network
+      - name: Checkout capsule
+        uses: actions/checkout@v2
+        with:
+          repository: vegaprotocol/vegacapsule
+          ref: main
+          token: ${{ secrets.VEGA_CI_BOT_GITHUB_TOKEN }}
+          path: './capsule'
+
+      #######
+      ## Build binaries
+      #######
+      - name: Build capsule
+        run: go install
+        working-directory: capsule
+
+      - name: Set GOBIN
+        run: echo GOBIN=$(go env GOPATH)/bin >> $GITHUB_ENV
+
+      - name: Install binaries
+        run: vegacapsule install-bins
+        env:
+          GITHUB_TOKEN: ${{ secrets.VEGA_CI_BOT_GITHUB_TOKEN }}
+          GOBIN: ${{ env.GOBIN }}
+
+      ######
+      ## Start capsule
+      ######
+      - name: Login to docker
+        run: echo -n ${{ secrets.VEGA_CI_BOT_GITHUB_TOKEN }} | docker login https://ghcr.io -u vega-ci-bot --password-stdin
```

Comment:
chore (non-blocking): we can kill this now the containers are public

Reply:
https://github.com/vegaprotocol/frontend-monorepo/issues/781